### PR TITLE
Add Messente to Developers page

### PR DESCRIPTION
**Pull request by jbaanus**
Messente provides API's for adding 2FA on your website with SMS or software token with Verigator app.
Logins to Messente Dashboard also support 2FA with SMS and Verigator.

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -251,6 +251,14 @@ websites:
       software: Yes
       doc: https://www.mapbox.com/help/two-step-verification/
 
+      name: Messente
+      url: https://messente.com/
+      img: messente.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://messente.com/documentation/verification-api
```
Comment (mxxcon) [inline _data/developer.yml]: @jbaanus it seems like your `doc` link describes how to use 2fa api, not how to enable 2fa for the users of messente service.

**Comment by jbaanus** [comment]
Removed the link, that was not helpful for enabling 2FA in Messente

File: _data/developer.yml (hunk 1 above)
Comment (jbaanus) [reply _data/developer.yml]: Updated the PR and removed the link to documentation.

**Review by stephengroat — APPROVED** [review]
thanks for the pr @jbaanus

**Comment by jbaanus** [comment]
Hey @mxxcon - seems like this PR is pending for your review.
Could you please check if the changes I made after your last comment are good to go? Thanks!

**Comment by Carlgo11** [comment]
Unfortunately messente.com is ranked >200K on alexa (currently above 1M) and as such it doesn't meet our requirements for new site additions (See our [guidelines](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines)).
Sorry for the inconvenience!

**Pull request closed without merging**